fb.d.ts

```diff
@@ -0,0 +1,61 @@
+import Promise = require("any-promise");
```

Comment:
Additionally, 2.0 of this library supports only NodeJS environments that support `Promise`-s natively ;)